packages/devtools_app/lib/src/screens/memory/framework/memory_controller.dart

```diff
@@ -65,26 +79,85 @@ class MemoryController extends DisposableController
     profile.dispose();
   }
 
-  DiffPaneController _createDiffController() =>
-      DiffPaneController(HeapGraphLoaderRuntime(chart.memoryTimeline));
+  Future<void> _init(
+    @visibleForTesting DiffPaneController? connectedDiff,
+    @visibleForTesting ProfilePaneController? connectedProfile,
+  ) async {
+    assert(!_initialized.isCompleted);
+    switch (_mode) {
+      case DevToolsMode.disconnected:
+        // TODO: load memory screen in disconnected mode, https://github.com/flutter/devtools/issues/6972
```

Comment:
TODO format (here and elsewhere in this PR) should be `TODO(username OR github issue)`. This will trigger lints in g3 if these are not formatted correctly.